### DISCUSSION TITLE
support for interactive operation request

### DISCUSSION THREAD

**Discussion by fujohnwang** [madflojo/efs2#5] [Ideas]
in the process of command execution, sometimes input is required, like the password or something else,  currently, efs2 may NOT handle such situation, maybe  we can support this feature in the near future to make this tool great 

![image](https://user-images.githubusercontent.com/451506/37870837-dc8aec6c-3011-11e8-878a-8ba7b2100bb5.png)

**Comment by madflojo** [comment]
It would be awesome to be able to add a flag to allow for interactive input as needed. I'd have to think about how possible this really is.
